includes/header.php

```diff
@@ -198,6 +140,18 @@
                                 </div>
                             </a>
 
+                            <a class="nav-link">
+                                <div class="sb-nav-link-icon">
+                                    <button type="button" class="btn btn-outline-primary" onclick="loadDemo()">
+                                        <!-- Add upload svg icon -->
+                                        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-file-earmark-arrow-up-fill" viewBox="0 0 16 16">
+                                        <path d="M9.293 0H4a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h8a2 2 0 0 0 2-2V4.707A1 1 0 0 0 13.707 4L10 .293A1 1 0 0 0 9.293 0M9.5 3.5v-2l3 3h-2a1 1 0 0 1-1-1M6.354 9.854a.5.5 0 0 1-.708-.708l2-2a.5.5 0 0 1 .708 0l2 2a.5.5 0 0 1-.708.708L8.5 8.707V12.5a.5.5 0 0 1-1 0V8.707z"/>
+                                        </svg>

```

Comment:
I would probably use pre-made icons instead of SVGs for this. What if you want to change the style later? The `path` data is not really tweakable by humans.

You can use those, for example: https://fontawesome.com/

Reply:
Done